Documentation/saml-connector.md

```diff
@@ -86,6 +86,12 @@ connectors:
     #     urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
     #
     nameIDPolicyFormat: persistent
+
+    # Optional: Use HTTP redirect binding instead of HTTP-POST binding
+    #
+    # Some identity provider only supports HTTP redirect binding
+    #
+    # redirectBinding: true
```

Comment:
this should probably be something like

```
binding: redirect # defaults to post
```

in case we want to support more bindings in the future.